src/S3.ts

```diff
@@ -170,7 +170,29 @@ class S3Bucket {
             return false
         }
     }
-
+    async containsCloudFrontLogsForYesterday(distributionId: string) {
```

Comment:
```suggestion
    async contentsUpdatedWithin(params)
```

where params would be something like:
```
{
  periodHours: int,
  keyPrefix: string
}
```